From 2e37cf5ba9499ae99d86f017ecb9cf72a206a022 Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Mon, 24 Mar 2014 11:58:16 -0400
Subject: [PATCH] Create and use libvirtd group for package install

For the nova-kvm package install, we need to create the libvirtd group
and add the nova user to the group. This is because the element now uses
its own libvirtd.conf which is configured to use the libvirtd group.

Change-Id: Ic345b76a17ff75b468da8a493c6bf856d730a586
---
 elements/nova-kvm/install.d/nova-package-install/80-nova-kvm | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/elements/nova-kvm/install.d/nova-package-install/80-nova-kvm b/elements/nova-kvm/install.d/nova-package-install/80-nova-kvm
index c68b0e2..22f8f7c 100755
--- a/elements/nova-kvm/install.d/nova-package-install/80-nova-kvm
+++ b/elements/nova-kvm/install.d/nova-package-install/80-nova-kvm
@@ -3,3 +3,6 @@
 set -eux
 
 install-packages openstack-nova-novncproxy
+
+groupadd libvirtd || true
+usermod -a -G libvirtd nova
-- 
1.8.5.3

